sdk/python/symbolchain/facade/NemFacade.py

```diff
@@ -37,20 +39,40 @@ def _create_nem_type_to_property_mapping():
 			PublicKey: 'public_key'
 		}
 
+	@staticmethod
+	def _to_non_verifiable_transaction(transaction):
+		class_name = type(transaction).__name__
+		if not class_name.startswith('NonVerifiable'):
+			class_name = 'NonVerifiable' + class_name
+
+		catbuffer_module = importlib.import_module('symbolchain.nc')
+		cls = getattr(catbuffer_module, class_name)
+
+		non_verifiable_transaction = cls()
+		for key in dir(non_verifiable_transaction):
+			if key.startswith('_') or key[0].isupper() or key in ['size', 'serialize', 'deserialize']:
```

Comment:
why is upper check?

Reply:
to quickly skip `TRANSACTION_VERSION`, `TRANSACTION_TYPE`, `TYPE_HINTS` (and any other potential properties)

Reply:
please add comment